modules/aws/volume-access/main.tf

```diff
@@ -0,0 +1,112 @@
+data "aws_caller_identity" "current" {}
+locals {
+  account_id                = data.aws_caller_identity.current.account_id
+  external_id               = (var.external_id != "" ? [{ test : "StringEquals", variable : "sts:ExternalId", values : [var.external_id] }] : [])
+  assume_conditions         = concat(local.external_id, local.source_identity, local.principal_check, local.vendor_federation)
+  support_assume_conditions = concat(local.external_id, local.source_identity)
+  source_identity           = (length(var.source_identities) > 0 ? [{ test : var.source_identity_test, variable : "sts:SourceIdentity", values : var.source_identities }] : [])
```

Comment:
We can remove this since it's tech debt from vendor-access module.

```suggestion
```

Reply:
Fixed